Runtime/Operators/TextInputOperator.cs

```diff
@@ -33,6 +32,16 @@ IRandomString randomString
                 return;
             }
 
+            var annotation = component.gameObject.GetComponent<InputFieldAnnotation>();
+            if (annotation != null)
+            {
+                // Overwrite rule if annotation is attached.
+                randomStringParams = _ => new RandomStringParameters(
+                    annotation.minimumLength,
+                    annotation.maximumLength,
+                    annotation.charactersKind);
+            }
+
```

Comment:
Could you make an injection point for the selecting character kind strategy like `ScreenPointStrategy`? Because some of my projects cannot use any annotations to make cheating hard. I can use test-helper.monkey for this project if the injection point exists.

Reply:
I think unifying strategy specification approach is better.

```csharp
public static DefaultTextInputStrategy
{
    public static RandomStringParams CreateRandomStringParams(GameObject gameObject)
    {
        if (component.gameObject.TryGetComponent<InputFieldAnnotation>(out var annotation)
        {
            return new RandomStringParameters(annotation.minimumLength, annotation.maximumLength, annotation.charactersKind);
        }
        return RandomStringParameters.Default;
    }
}
```

For example, we can extend by using it like:

```csharp
RandomStringParams = (gameObject) => HasMyAnnotation(gameObject) ? ParamsYouWant : DefaultTextInputStrategy(gameObject);
```

Another example, ignoring annotations can be achieved by the following code (I don't know how it is useful, but it can):

```csharp
RandomStringParams = (gameObject) => RandomStringParameters.Default;
```

Reply:
In the future, I plan to obsolete Strategies and allow the Operators to be defined by the user. (see #54)
Can I consider it at that time?
(This logic will be implemented in the Operator, maybe)